autogpt/memory/milvus.py

```diff
@@ -20,27 +20,42 @@ def __init__(self, cfg) -> None:
             cfg (Config): Auto-GPT global config.
         """
         # connect to milvus server.
-        connections.connect(address=cfg.milvus_addr)
+        if hasattr(self, 'use_zilliz_cloud'):
```

Comment:
suggest using a common method to create the connection for milvus and zilliz, it is so wired to check whether has use_zilliz_cloud attribute within Milvus object.

Reply:
updated, general code in zilliz and milvus has been extract into init_collection function in new commit.